lib/modules/showImages.js

```diff
@@ -72,14 +72,14 @@ module.options = {
 	},
 	maxWidth: {
 		type: 'text',
-		value: '640',
-		description: 'Max width of image displayed onscreen (enter zero for unlimited).',
+		value: '100%',
+		description: 'Max width of media (in pixels, enter zero for unlimited). Percentage of window width may also be used (e.g. "100%").',
 		advanced: true,
 	},
 	maxHeight: {
 		type: 'text',
-		value: '0',
-		description: 'Max height of image displayed onscreen (enter zero for unlimited).',
+		value: '95%',
```

Comment:
We should have migrators for these; you can add another one to the end of the array and remove [the existing one](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/ba1e2a871721245776cad1d7a35e2add4725eaf9/lib/core/migrate.js#L240).